### PR TITLE
[UI enhancements] 

### PR DESCRIPTION
**Pull request by kushal-arora-fw**
Please refer commit messages for multiple fixes.

### CODE REVIEW

File: apps/api/src/database/scripts/seed-service.ts [hunk 1]
```diff
@@ -119,7 +119,7 @@ export class SeedService {
             action = cleanText(action);
             if (action.includes('y')) {
               allowedAction = Permissions.PERFORM;
-            } else if (action.includes('l')) {
+            } else if (action.includes('lc')) {
```
Comment (dbayly-freshworks) [inline apps/api/src/database/scripts/seed-service.ts:122]: Nit - In future, for readability it might be good to replace this with a constant.

Comment (kushal-arora-fw) [reply apps/api/src/database/scripts/seed-service.ts:122]: it is a seed service, explicitly done to cater to client mistakes.

File: apps/web/src/common/constants.ts [hunk 2]
```diff
@@ -29,11 +28,11 @@ export const sidebarNavItems: SidebarButtonProps[] = [
...
       },
-      {
-        id: '004',
-        text: 'All Plan',
-        active: false,
-      },
+      // {
+      //   id: '004',
+      //   text: 'All Plan',
+      //   active: false,
```
Comment (dbayly-freshworks) [inline apps/web/src/common/constants.ts:34]: Nit - Best remove commented out code

File: apps/api/src/migration/1696529645524-updatePermissionsEnum.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class updatePermissionsEnum1696529645524 implements MigrationInterface {
+  name = 'updatePermissionsEnum1696529645524';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TYPE "public"."allowed_activity_permission_enum" RENAME TO "allowed_activity_permission_enum_old"`,
```
Comment (dbayly-freshworks) [inline apps/api/src/migration/1696529645524-updatePermissionsEnum.ts:8]: Nit - I have mixed feelings about having and old/new version of a column. Better practice would be to drop or update the column, and migrate the data as needed.

**Review by dbayly-freshworks — APPROVED** [review]
Two nits, otherwise no red flags

**Pull request merged**